lang/en.json

```diff
@@ -144,8 +144,8 @@
         "or": "or",
         "compare": "Compare",
         "sign_up": "Sign Up",
-        "login": "Login",
-        "logout": "Logout",
+        "login": "Log in",
```

Comment:
This is debatable and depends on context. If the context is to take an action ("present your credentials and authenticate"), then it's correct as "Login". If it's the result of taking the action, then it's fine. However, it's honestly probably better to say "Sign in" since that's correct no matter the context.